changelog.txt

```diff
@@ -4,6 +4,7 @@
 * Fix - Fix hover dialog for close button on modals, unify styling and layout of modal buttons.
 * Update - Use Site Language when rendering Stripe elements.
 * Update - Use blog ID for authenticating most of the requests.
+* Add - Redirect to WooCommerce home page after successful WooCommerce Payments KYC
```

Comment:
Can we expand the abbreviation here, so external folks who will read the changelog have an idea of what is this about?

Reply:
Good idea!